server/src/entity/Account.ts

```diff
@@ -46,6 +48,15 @@ export class Account extends BaseEntity {
   })
   public isActive: boolean;
 
-  @OneToMany((type) => Shift, (shift) => shift.server)
+  @OneToMany(() => Shift, (shift) => shift.server)
   public shifts: Shift[];
+
+  @OneToMany(() => Review, (review) => review.reviewer)
+  public reviews: Review[];
+
+  @ManyToMany(() => Restaurant, (restaurant) => restaurant.managers)
```

Comment:
is this a reasonable expectation that one person can manage many restaurants?

Reply:
+ Invalid syntax `@ManyToOne(type => User, user => user.photos)`

Reply:
Yes, if you are the boss of a chain store. You should be able to see many restaurants.

Reply:
> * Invalid syntax `@ManyToOne(type => User, user => user.photos)`

what??? where is this, I don't see it

Reply:
http://typeorm.io/#/many-to-one-one-to-many-relations

Reply:
but not invalid

Reply:
well, i would agree; however, I would stick with the documented behavior so we don't have any unexpected bugs

Reply:
wait... you wrote your in the same way! Do we just change all then?